docs/pipeline_config_documentation/pipeline_ingestion_yml.md

```diff
@@ -0,0 +1,494 @@
+    In case you have cell level metadata such as results from a demultiplexing algorithm, you can incorporate it into the MuData object.
+    The demultiplexing results should be stored in one csv file containing 2 columns, barcode_id, and sample_id.
+    Note that the sample_id should match the sample_id column in the submission file.
+  
+    * <p class="parameter">include</p> Boolean, Default: True<br>
+        <p>Set to True if you want to include barcode level data.
+        </p><br>
+      
+    * <p class="parameter">path</p> String
+        <p>Insert path to the csv file containing the demultiplexing results.
+           Requires include to be set to True.
+        </p><br>
+      
+    * <p class="parameter">metadatacols</p> String
+        <p>Provide the metadata columns in the file specified via path that should be incorporated in the MuData object.
+        </p><br>
+    
+
+### Loading prot data - additional options
+* <p class="parameter">protein_metadata_table</p> String
+    
+    File containing additional information on the proteins.
+    By default, the ingestion workflow will use the first column of the cellranger features.tsv.gz.
+    If you want to add additional information about the antibodies (e.g. whether they are hashing antibodies or isotypes), you need to provide an additional table containing these information.
+    The first column of the table must be equivalent to the first column of cellrangers features.tsv.gz and it must be stored as a txt file.
+    For example, you could create a table that include a column called isotype (containing True or False) to run QC isotypes, and a column called hashing_ab (containing True or False) if your dataset contains hashing antibodies which you wish to split into a separate modality.
+
+
+* <p class="parameter">index_col_choice</p> String
+    
+    Column name from protein_metadata_table that should be included in the MuData object.
+    If you want to update the MuData object storing all our data with a column from the table specified in protein_metadata_table, specify the column name here.
+    Ensure that there are no overlaps with the gene symbols used for the rna modality, otherwise you might face issues down the line.
+    If there are overlaps with the rna gene symbols, then you might have trouble down the line
+    Consider adding e.g. a prefix to make the protein labels unique (e.g. "prot_CD4").
+ 
+
+* <p class="parameter">load_prot_from_raw</p> Boolean, Default: False
+    
+    In case separate prot and rna 10X outputs are provided, the pipeline will load the filtered rna 10X outputs and the raw prot 10X counts.
+
+    
+* <p class="parameter">subset_prot_barcodes_to_rna</p> Default: False
+    
+    Set to True if you want to treat filtered rna barcodes as the "real" cells (which we assume one wants to do in most cases), and, additionally, the out prot assay barcodes should mathc the rna ones.
```

Comment:
if providing separate prot and rna 10X output ("load_prot_from_raw": True) , set  subset_prot_barcodes_to_rna to True if ...